apps/webcomponents/src/app/components/base.component.ts

```diff
@@ -68,13 +83,35 @@ export class BaseComponent implements OnChanges, OnInit {
       this.titleFont
     )
     this.facade.init(this.searchId)
+    this.copyFontFacesToDocument()
```

Comment:
Wow, good finding, weird that I have the feeling to already have seen icons in the web components in the past.

Reply:
it used to work in the demo app because the icons font was  present in the host page